backend/package.json

```diff
@@ -29,9 +33,14 @@
     "passport": "^0.4.1",
     "passport-local": "^1.0.0",
     "passport-local-mongoose": "^6.0.1",
+    "prettier": "^2.3.2",
```

Comment:
these packages (eslint configs and prettier) should have been in devDependencies.